chainer/_version.py

```diff
@@ -13,7 +13,7 @@
             'cupy-cuda80',
             'cupy',
         ],
-        'specifier': '==7.0.0a1',
+        'specifier': '==7.0.0b1',
```

Comment:
Please fix this as well.
It seems you did conflict resolution in a wrong way (there are unrelated commits in the history).
Could you fix that (I guess squash is the easiest way)?